### ISSUE TITLE
 [Experiment] The text field from the share modal has straight corners instead of rounded

### ISSUE THREAD

**Issue by heres-maria** [mozilla/blurts-server#1799]
**[Affected Versions]:**
- Firefox Beta 79.0 (Build ID: 20200720193547)
- Chrome 84.0.4147.89

**[Affected Platforms]:**
- Windows 10
- Linux Mint 19.2

**[Prerequisites]:**
- You are on the [Treatment](https://monitor-v2.herokuapp.com/?experimentBranch=vb) branch.

**[Steps to reproduce]:**                                        
1. Click the “Share Monitor” button from the footer.
2. Observe the corners of the text field and the "Copy" button.

**[Expected results]:**
- The corners of the text field and button are rounded.

**[Actual results]:**
- The corners of the text field and button are not rounded.

**[Note]:**
- This is also reproducible for the Specific Breach modal.
- This issue is not reproducible on mobile devices or small sized windows on desktop.

![image](https://user-images.githubusercontent.com/53752296/88405885-8bc82080-cdd8-11ea-8e60-25129c2b8652.png)

**Comment by maxxcrawford** [comment]
Pulling a screenshot from the Sketch file by @changecourse. Looks rounded. Thanks! 

<img width="564" alt="image" src="https://user-images.githubusercontent.com/2692333/88406214-fb68ec00-cd95-11ea-92df-8133a0931886.png">

**Comment by heres-maria** [comment]
I have verified that the issue is no longer reproducible.
The corners are now rounded for the desktop version as well - both the General and Specific modals.
Tested using Firefox Nightly 80.0a1(Linux Mint 19.2, Windows 10), Firefox Beta 79.0(Windows 10, Linux Mint 19.2), Firefox Release 78.0.2(Windows 10), Firefox for Android 68.10.1 (Android 10), Firefox for iOS 27.0 (iOS 12.4.6).
As a result, I am closing this issue.
